billing/views.py

```
@@ -0,0 +1,277 @@
+"""
+Views for billing app.
+"""
+
+from rest_framework import status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from django.db import transaction as db_transaction
+from django.shortcuts import get_object_or_404
+from decimal import Decimal
```

Ruff (0.12.2)

`decimal.Decimal` imported but unused

Remove unused import: `decimal.Decimal`

(F401)